manifests/kustomize/base/installs/multi-user/pipelines-profile-controller/sync.py

```diff
@@ -26,7 +26,8 @@
 # To interact with seaweedfs user management. Region does not matter.
 iam = session.create_client('iam', region_name='foobar')
 # S3 client for lifecycle policy management
-s3 = session.create_client('s3', region_name='foobar')
+s3_endpoint_url = os.environ.get("S3_ENDPOINT_URL", "http://seaweedfs:8333")
```

Comment:
Please try out http://seaweedfs.kubeflow:8333 . See https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#namespaces-of-services